### ISSUE TITLE
args cant overide

### ISSUE THREAD

**Issue by iris-qq** [qqwweee/keras-yolo3#313]
`optional arguments:
  -h, --help         show this help message and exit
  --model MODEL      path to model weight file, default model_data/yolo.h5
  --anchors ANCHORS  path to anchor definitions, default
                     model_data/yolo_anchors.txt
  --classes CLASSES  path to class definitions, default
                     model_data/coco_classes.txt`
this args help to start yolo_video.py. but i found in yolo.py,author want to overrides
`def __init__(self, **kwargs):
        self.__dict__.update(self._defaults)  # set up default values
        self.__dict__.update(kwargs)  # and update with user overrides
        self.class_names = self._get_class()`
but they have different key ,is this a bug?

**Comment by AryaCao** [comment]
yes, i change the args key from "--model'' to "--model_path", and it can override .
like this:
![args](https://user-images.githubusercontent.com/34637295/52995027-6bae6800-3454-11e9-8bf8-9a72947fe470.png)
![args1](https://user-images.githubusercontent.com/34637295/52995046-7963ed80-3454-11e9-882e-457aaf1b94cf.png)
